.claude/commands/index.md

```
@@ -0,0 +1,96 @@
+Commands: `/cmd --flags` | 19 total | Universal flags available
+
+@include shared/flag-inheritance.yml#Universal_Always
+
+## Command Categories
+**Analysis**: `/analyze` `/scan` `/explain` `/review`
+**Build**: `/build` `/deploy` `/migrate` 
+**Manage**: `/task` `/load` `/cleanup`
+**Dev**: `/test` `/troubleshoot` `/improve`
+**Utils**: `/design` `/document` `/estimate` `/dev-setup` `/git` `/spawn`
+
+## Commands w/ Primary Flags
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)